provision-osx/defaults/main.yml

```diff
@@ -4,6 +4,11 @@
 
 remote_tmp_dir: /tmp
 
+git_install_url: https://sourceforge.net/projects/git-osx-installer/files/git-2.13.1-intel-universal-mavericks.dmg/download?use_mirror=autoselect
```

Comment:
Taken a look, this URL seems the best choice. May have some issue with productisation though